components/base/DPersonCard.vue

```diff
@@ -28,9 +28,9 @@
     </template>
     <template #footer>
       <footer class="py-3 px-3">
-        <h2>{{ name }}</h2>
+        <p class="person-subtitle">{{ name }}</p>
```

Comment:
```suggestion
        <h5 >{{ name }}</h5>
```
might need the `content` class